parley/src/style/mod.rs

```diff
@@ -22,6 +22,23 @@ pub enum WhiteSpaceCollapse {
     Preserve,
 }
 
+/// Line height, specified either in absolute units or relative to the font
+/// size. You can create one from an [`f32`] via [`From`]/[`Into`], which will
+/// result in a relative line height.
+#[derive(Debug, Clone, Copy, PartialEq)]
+pub enum LineHeight {
+    /// Line height specified as a multiple of the font size.
+    Relative(f32),
+    /// Line height specified in absolute units.
+    Absolute(f32),
+}
+
+impl From<f32> for LineHeight {
+    fn from(value: f32) -> Self {
+        Self::Relative(value)
+    }
+}
+
```

Comment:
I was thinking `From<LineHeight> for StyleProperty`; `From<f32> for LineHeight` is ambiguous.